docs/ops/shape/Reshape_1.md

```diff
@@ -8,40 +8,41 @@
 
 **Detailed description**:
 
-*Reshape* layer takes two input tensors: the tensor to be resized and the output tensor shape. The values in the second tensor could be -1, 0 and any positive integer number. The two special values -1 and 0:
+*Reshape* takes two input tensors: `data` to be resized and `shape` of the new output. The values in the `shape` could be `-1`, `0` and any positive integer number. The two special values `-1` and `0`:
    * `0` means "copy the respective dimension *(left aligned)* of the input tensor" if `special_zero` is set to `true`; otherwise it is a normal dimension and is applicable to empty tensors.
-   * `-1` means that this dimension is calculated to keep the overall elements count the same as in the input tensor. Not more than one `-1` can be used in a reshape operation. 
+   * `-1` means that this dimension is calculated to keep the overall elements count the same as in the input tensor. Not more than one `-1` can be used in a reshape operation.
 
-If `special_zero` is set to `true` index of `0` cannot be larger than the rank of the input tensor. 
+If `special_zero` is set to `true` index of `0` cannot be larger than the rank of the input tensor.
 
 **Attributes**:
 
 * *special_zero*
 
-  * **Description**: *special_zero* controls how zero values in `shape` are interpreted. If *special_zero* is `false`, then 0 is interpreted as-is which means that output shape will contain a zero dimension at the specified location. Input and output tensors are empty in this case. If *special_zero* is `true`, then all zeros in `shape` implies the copying of corresponding dimensions from `data.shape` into the output shape *(left aligned)*.
+  * **Description**: *special_zero* controls how zero values in `shape` are interpreted. If *special_zero* is `false`, then `0` is interpreted as-is which means that output shape will contain a zero dimension at the specified location. Input and output tensors are empty in this case. If *special_zero* is `true`, then all zeros in `shape` implies the copying of corresponding dimensions from `data.shape` into the output shape *(left aligned)*.
   * **Range of values**: `false` or `true`
   * **Type**: boolean
   * **Default value**: None
   * **Required**: *yes*
 
 **Inputs**:
 
-*   **1**: `data` -- multidimensional input tensor of type *T*. *Required*.
+*   **1**: `data` - multidimensional input tensor of type *T*. **Required**.
```

Comment:
Recently instead of "_multidimensional input tensor of type *T*_", we write "_A tensor of type T and arbitrary shape_" to be explicit about the shape